hr_timesheet_sheet/models/hr_timesheet_sheet.py

```diff
@@ -257,10 +257,11 @@ def _compute_line_ids(self):
             dates = sheet._get_dates()
             if not dates:
                 continue
-            timesheets = sheet._get_timesheet_lines()
             lines = self.env['hr_timesheet.sheet.line']
             for date in dates:
-                for project in timesheets.mapped('project_id'):
+                timesheets = sheet._get_timesheet_lines()  # do not move this
```

Comment:
Otherwise you will get errors on tests and while using the module.

Reply:
Well, you should know that this will be just a temporal fix. It's not a correct fix, because you are doing a search for each date and it's inefficient, but just serves to bypass the errors produced by the other changes. This fix it's related to merged timesheets and having deleted records in self.timesheet_ids. In https://github.com/OCA/hr-timesheet/pull/188, when applying https://github.com/OCA/hr-timesheet/pull/188/commits/8ab95b7af177608ed055b70fc3a87088d6d03d01, then is consequently necessary apply https://github.com/OCA/hr-timesheet/pull/188/commits/3e7c2c78b8bd8551544b94134e19351c69f50865 where the correct fix applies.

Reply:
this can be minimized